src/components/YourCompany/NameTranslations/AddNameTranslation.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div>
+    <confirm-dialog
+      ref="confirmTranslationDialog"
+    />
+    <!-- Name Translation form -->
+    <v-form v-model="nameTranslationForm" class="name-translation-form">
+      <v-row>
+        <v-col class="pb-0">
+          <v-text-field
+            filled
+            persistent-hint
+            label="Name Translation"
+            id="name-translation-input"
+            v-model="nameTranslation"
+            :rules="nameTranslationRules">
+          </v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col>
+          <div class="action-btns">
+          <v-btn large color="primary"
+            id="name-translation-btn-ok"
+            :disabled="!nameTranslationForm"
+            @click="addTranslation"
+          >
+            OK
+          </v-btn>
+          <v-btn large outlined color="primary"
+            id="name-translation-btn-cancel"
+            @click="cancelNameTranslation"
```

Comment:
I prefer to see the event handler with its (), to reinforce the fact that it's a method (and in this case, takes no parameters).

(I know this is a valid short-hand but I'm trying to make reviewing easier ;) )